perllib/FixMyStreet/App/Controller/Around.pm

```diff
@@ -416,7 +416,10 @@ sub lookup_by_ref : Private {
             external_id => $ref
         ];
 
-    my $problems = $c->cobrand->problems->search( $criteria );
+    my $problems = $c->cobrand->problems->search({ -and => {
```

Comment:
Doesn't matter, but I don't think the `-and` is needed here; `search({ non_public => 0, -or => $criteria })` should be the same.